### PR TITLE
japanese translation of chapter 7 performance

### PR DESCRIPTION
**Pull request by MSakamaki**
Chapter 7: performance has been translated into Japanese.

### CODE REVIEW

File: src/content/ja/2019/performance.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
+---
+part_number: II
+chapter_number: 7
+title: パフォーマンス
+description: コンテンツの初回ペイント（FCP）、最初のバイトまでの時間（TTFB）、初回入力遅延（FID）を取り扱う2019 Web Almanac パフォーマンスの章。
+authors: [rviscomi]
+reviewers: [JMPerez,obto,sergeychernyshev,zeman]
+discuss: 1762
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2019-11-07T21:46:11.000Z
```
Comment (rviscomi) [inline src/content/ja/2019/performance.md]:
```suggestion
last_updated: 2019-11-17T00:00:00.000Z
```

Comment (bazzadp) [reply src/content/ja/2019/performance.md]: We should probably change the published date too. But can probably hold both off until we decide on a go live date for Japanese edition (whether all chapters or just some) and bulk change them all then?

Btw, while on the subject of the meta data, are we adding a translators line to meta data (and the page!) to give the translator credit and act as a primary contact point for translation issues? Should I raise an issue for this?

Comment (rviscomi) [reply src/content/ja/2019/performance.md]: > Btw, while on the subject of the meta data, are we adding a translators line to meta data (and the page!) to give the translator credit and act as a primary contact point for translation issues? Should I raise an issue for this?

Yes and yes! Good call.

Comment (bazzadp) [reply src/content/ja/2019/performance.md]: https://github.com/HTTPArchive/almanac.httparchive.org/issues/507 raised. Will have a go at fixing this myself but raised the issue so people can comment on design.

@MSakamaki will you add the necessary translator(s) meta data to this PR (while we work separately on actually using that meta data)?

Comment (MSakamaki) [reply src/content/ja/2019/performance.md]: Great job @bazzadp!
I will fix the translator and last_updated metadata as per #507 @rviscomi

**Pull request merged**